src/coreclr/jit/codegencommon.cpp

```diff
@@ -2002,14 +2002,6 @@ void CodeGen::genEmitMachineCode()
     compiler->compCodeGenDone = true;
 #endif
 
-#if defined(DEBUG) || defined(LATE_DISASM)
-    // Add code size information into the Perf Score
-    // All compPerfScore calculations must be performed using doubles
-    compiler->info.compPerfScore += ((double)compiler->info.compTotalHotCodeSize * (double)PERFSCORE_CODESIZE_COST_HOT);
-    compiler->info.compPerfScore +=
-        ((double)compiler->info.compTotalColdCodeSize * (double)PERFSCORE_CODESIZE_COST_COLD);
```

Comment:
You should remove the definitions of `PERFSCORE_CODESIZE_COST_HOT` / `PERFSCORE_CODESIZE_COST_COLD`, which will be no longer used.

Reply:
thanks, done.